### ISSUE TITLE
Training loss is nan

### ISSUE THREAD

**Issue by FudongGe** [fudan-zvg/BezierGS#12]
Great work! 

During training, the loss becomes NaN. Have you encountered this issue, or do you have any suggestions?

**Comment by xiao10ma** [comment]
Hi, thank you for your interest in our work!

I suspect the `NaN` issue might be happening because some dynamic objects only have a few frames available for optimization, or the objects themselves consist of very few points. This sparsity could be leading to instability during the optimization process.

Here are a couple of suggestions you could try:

1.  You could try extending the number of frames used for these objects to see if that resolves the problem.
2.  Alternatively, try deleting these sparse dynamic objects. Since they have very few points, they might be adequately represented by the static Gaussian primitives alone.

You can try the methods I suggested. If you have any further questions, feel free to ask me again.
